Index: mooveTravel/.nuxt/manifest/latest.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\"id\":\"dev\",\"timestamp\":1752569102406}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mooveTravel/.nuxt/manifest/latest.json b/mooveTravel/.nuxt/manifest/latest.json
--- a/mooveTravel/.nuxt/manifest/latest.json	(revision 276ffc49212391c6d753374a8559f61245cb1a7c)
+++ b/mooveTravel/.nuxt/manifest/latest.json	(date 1752586703911)
@@ -1,1 +1,1 @@
-{"id":"dev","timestamp":1752569102406}
\ No newline at end of file
+{"id":"dev","timestamp":1752586681328}
\ No newline at end of file
Index: mooveTravel/.nuxt/manifest/meta/dev.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\"id\":\"dev\",\"timestamp\":1752569102406,\"matcher\":{\"static\":{},\"wildcard\":{},\"dynamic\":{}},\"prerendered\":[]}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mooveTravel/.nuxt/manifest/meta/dev.json b/mooveTravel/.nuxt/manifest/meta/dev.json
--- a/mooveTravel/.nuxt/manifest/meta/dev.json	(revision 276ffc49212391c6d753374a8559f61245cb1a7c)
+++ b/mooveTravel/.nuxt/manifest/meta/dev.json	(date 1752586703921)
@@ -1,1 +1,1 @@
-{"id":"dev","timestamp":1752569102406,"matcher":{"static":{},"wildcard":{},"dynamic":{}},"prerendered":[]}
\ No newline at end of file
+{"id":"dev","timestamp":1752586681328,"matcher":{"static":{},"wildcard":{},"dynamic":{}},"prerendered":[]}
\ No newline at end of file
Index: mooveTravel/.nuxt/nuxt.d.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/// <reference types=\"@nuxt/devtools\" />\r\n/// <reference types=\"@nuxt/telemetry\" />\r\n/// <reference path=\"types/builder-env.d.ts\" />\r\n/// <reference types=\"nuxt\" />\r\n/// <reference path=\"types/app-defaults.d.ts\" />\r\n/// <reference path=\"types/plugins.d.ts\" />\r\n/// <reference path=\"types/build.d.ts\" />\r\n/// <reference path=\"types/schema.d.ts\" />\r\n/// <reference path=\"types/app.config.d.ts\" />\r\n/// <reference types=\"vue-router\" />\r\n/// <reference path=\"types/middleware.d.ts\" />\r\n/// <reference path=\"types/nitro-middleware.d.ts\" />\r\n/// <reference path=\"types/layouts.d.ts\" />\r\n/// <reference path=\"components.d.ts\" />\r\n/// <reference path=\"imports.d.ts\" />\r\n/// <reference path=\"types/imports.d.ts\" />\r\n/// <reference path=\"schema/nuxt.schema.d.ts\" />\r\n/// <reference path=\"types/nitro.d.ts\" />\r\n\r\nexport {}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mooveTravel/.nuxt/nuxt.d.ts b/mooveTravel/.nuxt/nuxt.d.ts
--- a/mooveTravel/.nuxt/nuxt.d.ts	(revision 276ffc49212391c6d753374a8559f61245cb1a7c)
+++ b/mooveTravel/.nuxt/nuxt.d.ts	(date 1752586686165)
@@ -1,5 +1,5 @@
-/// <reference types="@nuxt/devtools" />
 /// <reference types="@nuxt/telemetry" />
+/// <reference types="@nuxt/devtools" />
 /// <reference path="types/builder-env.d.ts" />
 /// <reference types="nuxt" />
 /// <reference path="types/app-defaults.d.ts" />
Index: mooveTravel/.nuxt/nitro.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"date\": \"2025-07-15T08:45:07.265Z\",\r\n  \"preset\": \"nitro-dev\",\r\n  \"framework\": {\r\n    \"name\": \"nuxt\",\r\n    \"version\": \"3.17.5\"\r\n  },\r\n  \"versions\": {\r\n    \"nitro\": \"2.11.12\"\r\n  },\r\n  \"dev\": {\r\n    \"pid\": 14608,\r\n    \"workerAddress\": {\r\n      \"socketPath\": \"\\\\\\\\.\\\\pipe\\\\nitro-worker-14608-1-1-978.sock\"\r\n    }\r\n  }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mooveTravel/.nuxt/nitro.json b/mooveTravel/.nuxt/nitro.json
--- a/mooveTravel/.nuxt/nitro.json	(revision 276ffc49212391c6d753374a8559f61245cb1a7c)
+++ b/mooveTravel/.nuxt/nitro.json	(date 1752586706275)
@@ -1,5 +1,5 @@
 {
-  "date": "2025-07-15T08:45:07.265Z",
+  "date": "2025-07-15T13:38:26.217Z",
   "preset": "nitro-dev",
   "framework": {
     "name": "nuxt",
@@ -9,9 +9,9 @@
     "nitro": "2.11.12"
   },
   "dev": {
-    "pid": 14608,
+    "pid": 10356,
     "workerAddress": {
-      "socketPath": "\\\\.\\pipe\\nitro-worker-14608-1-1-978.sock"
+      "socketPath": "\\\\.\\pipe\\nitro-worker-10356-1-1-9042.sock"
     }
   }
 }
\ No newline at end of file
